### ISSUE TITLE
[TODO] upgrade to bokeh v3

### ISSUE THREAD

**Issue by cjdoris** [cjdoris/Bokeh.jl#17]
It's not released yet, but there is a branch for bokeh v3, so I should make a corresponding branch to try upgrading.

**Comment by cjdoris** [comment]
Yeah but I've got limited time for this package, help welcome. I started the migration but don't remember how far I got.

**Comment by RoyiAvital** [comment]
What are the chances of this to happen?
It would be great.

**Comment by cjdoris** [comment]
Afraid nothing has changed since my previous post - my limited time is on other projects. If you want to tackle it I'd be happy to guide you.

**Comment by JamesWrigley** [comment]
I'm kinda interested in this and might have some time to help out, are there any docs describing how to update the package?

**Comment by JamesWrigley** [comment]
Alrighty, thanks 👍 I'll give that a shot at some point.
